hook/hooktest/service.go

```diff
@@ -20,22 +20,15 @@ import (
 // It should be buffered with a size of at least 2.
 func NewServiceFunc(r *Runner, notify chan ServiceEvent) func(service.OSServiceParams) service.OSService {
```

Comment:
They're mockable interfaces that (at the moment) are used to control upstart services.